src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.ts

```diff
@@ -136,6 +156,33 @@ export class DragAndDropQuestionEditComponent implements OnInit, OnChanges, Quiz
         }
     }
 
+    ngAfterViewInit(): void {
+        if (this.question.backgroundFilePath) {
+            this.backgroundFilePath = this.question.backgroundFilePath;
+            // Trigger image render with the question background file path in order to adjust the click layer.
+            setTimeout(() => {
+                this.changeDetector.detectChanges();
+            }, 0);
+        }
+
+        this.backgroundImage.endLoadingProcess
+            .pipe(
+                filter((x) => x === ImageLoadingStatus.SUCCESS),
```

Comment:
I don't like the variable name `x` --> please do not use them and instead use more descriptive variable names.
(please fix it in a follow-up)